src/Error/ExceptionTrap.php

```diff
@@ -98,7 +98,11 @@ public function renderer(Throwable $exception, $request = null)
 
         /** @var class-string|callable $class */
         $class = $this->_getConfig('exceptionRenderer');
-        if (!$class) {
+        if (!$class || ($class === ExceptionRenderer::class && PHP_SAPI === 'cli')) {
+            // Default to detecting the exception renderer if we're
+            // in a CLI context and the Web renderer is currently selected.
+            // This indicates old configuration or user error, in both scenarios
+            // it is preferrable to use the Console renderer instead.
             $class = $this->chooseRenderer();
```

Comment:
I think we should throw a deprecation warning here.

Reply:
Added now.